src/redux/configurestore.js

```diff
@@ -0,0 +1,12 @@
+import { createStore, combineReducers } from 'redux';
+import bookReduce from './books/books';
+import categReduce from './categories/categories';
+
+const rootReducer = combineReducers({
+  book: bookReduce,
+  categ: categReduce,
+});
+
+const store = createStore(rootReducer);
+
+export default store;
```

Comment:
- [ ] You are to make use of Redux Toolkit and its methods to configure your store using the `configureStore`. It should be imported from the redux toolkit and then used to f=configure your store. Kindly consider working with the project requirement.

**Below is the project requirement that required the use of Toolkit**
![image](https://user-images.githubusercontent.com/50941074/192912664-1e3490e5-2823-4167-902a-f95633840cfb.png)

Consider checking the [Redux Toolkit guide](https://redux-toolkit.js.org/api/configureStore) on how to setup your store.